src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs

```diff
@@ -355,7 +355,8 @@ type GoToDefinitionFilter(textDocument: ITextDocument,
                         | String.StartsWith "https://raw.github.com" _-> formattedGithubUrl
                         | String.StartsWith "https://github.com" _-> formattedGithubUrl
                         | String.StartsWith "https://bitbucket.org" _-> 
-                            sprintf "%s#cl-%d" (url |> replace "/raw/" "/src/") r.StartLine
+                            let fileName = Path.GetFileName r.FileName
+                            sprintf "%s?fileviewer=file-view-default#%s-%d" (url |> replace "/raw/" "/src/") fileName r.StartLine
```

Comment:
Bitbucket source urls with line number looks like this: 
> https://bitbucket.org/IntelliFactory/websharper.google.maps/src/45a6136882edfecd168e0c43c17eced387890be4/build.cmd?at=default&fileviewer=file-view-default#build.cmd-7

This looks like what the code does to me. Except the branch is missing (`?at=default`). I haven't tested the code yet.